src/odemis/util/filename.py

```diff
@@ -198,6 +198,27 @@ def generate_fn(ptn, count):
 
     return os.path.join(path, fn + ext)
 
+def individualize_filename(fn, names):
```

Comment:
I'm not super convinced by "individualize", it was quite unclear what it'd do, when reading the caller text. Maybe `make_unique_filename()`. 

Also, I'd recommend "names" -> "existing".